build.gradle

```diff
@@ -0,0 +1,53 @@
+plugins {
+	id 'idea'
```

Comment:
Is use of tabs here is a choice for a specific reason? If so I'd be interested to know the reasoning behind it. In my experience I try to avoid tabs as they have different settings in different systems so their display can't be controlled by the code author.